doc/base/classes.xml

```diff
@@ -423,6 +613,13 @@
 			<description>
 				Parse JSON text to a Variant (use [method typeof] to check if it is what you expect).
 				Be aware that the JSON specification does not define integer or float types, but only a number type. Therefore, parsing a JSON text will convert every numerical values to [float] types.
+				[codeblock]
+				p = parse_json('["a", "b", "c"]')
+				if typeof(p) == TYPE_ARRAY:
+				    print(p[0])
+				else:
+					print("unexpected results")
```

Comment:
The last tab should be spaces. ;)

Reply:
doh I thought I caught them all. ill fix it up. :)